include/mimalloc/internal.h

```diff
@@ -115,6 +115,8 @@ size_t     _mi_os_large_page_size(void);
 
 void*      _mi_os_alloc_huge_os_pages(size_t pages, int numa_node, mi_msecs_t max_secs, size_t* pages_reserved, size_t* psize, mi_memid_t* memid);
 
+void*      mi_align_up_ptr(void* p, size_t alignment);
```

Comment:
This in particular is suspicious as the function doesn't start with `_` but I wasn't sure what the real intent is, it seems to be a function only used int he wasi prim currently